src/js/midi/helpers.js

```diff
@@ -1,5 +1,16 @@
-const getNameFromPort = (port) => `${port.name} (version ${port.version}) ${port.manufacturer}`
+const getNameFromPort = (port) => {
+  const { id, name, version, manufacturer } = port
+  return `${name} (version ${version}) ${manufacturer} (${id})`
```

Comment:
The `id` part is dynamically generated information that's not useful for the user and at least on my machine the id strings are very long and mess up the UI.

Reply:
Unfortunately I use `getNameFromPort` to keep track of pressed notes not just for the display, so I need to refactor a bit to work around this